lib/app.js

```diff
@@ -54,6 +54,7 @@ module.exports = settings => {
     if (isNaN(req.establishmentId)) {
       return next(new NotFoundError());
     }
+    res.locals.static.establishmentId = req.establishmentId; // needed for breadcrumbs
```

Comment:
does this not get added to locals? I'm pretty sure we spread req.params to locals.static in the page router in service

Reply:
I think it depends what route you hit - as the projectVersion routes don't seem to have it.

Reply:
projectVersion edit routes in asl, or view in pages? or all of the above?